index.bs

```diff
@@ -5097,10 +5185,12 @@ interface AudioListener {
 <h4 id="AudioListener-attributes">
 Attributes</h4>
 
-For all of the following {{AudioParam}}s, the {{AudioParam}}
-rate is specified by the <dfn>listener AudioParam rate</dfn>
-which is <a>a-rate</a> when any connected {{PannerNode}} is
-<a>a-rate</a> and is <a>k-rate</a> otherwise.
+For all of the following {{AudioParam}}s, the {{AudioParam}} rate is
```

Comment:
I had to read this paragraph several times to understand what it means, and I am still not entirely sure.

Reply:
The original was kind of wrong and didn't really spell out what the listener rate was.  Now we are more explicit and say the listener is a-rate if any panner node has an audio param with an effective automation rate of a-rate.  "effective rate" means the k-rate if the panner is HRTF.  Otherwise, it is the automationRate for the param.

Reply:
After much discussion off-line, we've agreed this is wrong and should be removed.  Instead, in the `PannerNode`, we should say that if the `panningModel` is "HRTF", the actual automation rate for the `PannerNode` and `AudioListener` is ignored and use "k-rate".  Otherwise, the actual selected automation rate is used.